docs/ldml/tr35-keyboards.md

```diff
@@ -1953,11 +1954,11 @@ The following are additions to standard Regex syntax.
 
     In this usage, the variable with `id="zwnj"` will be substituted in at this point in the expression. The variable can contain a range, a character, or any other portion of a pattern. If `zwnj` is a simple string, the pattern will match that string at this point.
 
-- **Set and UnicodeSet variables**
+- **Set or uSet variables**
```

Comment:
If we go with lower casing, then:

```suggestion
- **Set or uset variables**
```

or

```suggestion
- **Uset or set variables**
```

Reply:
I'm going to go with lowercase but monospace